packages/utils/src/test-utils/validateValueWithinRange.ts

```diff
@@ -0,0 +1,8 @@
+export const validateValueWithinRange = (params: { value: number; min: number; max: number }) => {
```

Comment:
I believe `expectToBeInRange` fits better here, considering this is meant to be an _assertion_ utility.

Tests can be written fluently:

```ts
  import { expectToBeInRange } from '...';

  it('should return true when value is within the range', () => {
    expectToBeInRange({ value: 7, min: 5, max: 10 });
  });
```

```suggestion
export const expectToBeInRange = (params: { value: number; min: number; max: number }) => {
```

Reply:
@arboleya makes sense